cura/MachineAction.py

```diff
@@ -114,3 +124,26 @@ def qmlPath(self) -> "QUrl":
     @pyqtSlot(result = QObject)
     def getDisplayItem(self) -> Optional["QObject"]:
         return self._createViewFromQML()
+
+    @pyqtSlot(result = bool)
+    def openAsDialog(self) -> bool:
```

Comment:
I feel that this should be a pyqtProperty and it should probably also be called something like "shouldOpenAsDialog", as openAsDialog feels more like a command to open an action (eg; Open this action as a dialog).

Reply:
Hi! you are right, it should called like that due is not an action. Im on it.